### PR TITLE
update push-sdk(swift) download links

### PR DESCRIPTION
**Pull request by cvasilak**
@corinnekrych @matzew mind to view?

### CODE REVIEW

**Comment by matzew** [comment]
what about the ObjC? or link to Cocapods?

**Comment by cvasilak** [comment]
@matzew sure I've added it and goes to the cocoapod query ?AeroGear-Push so both the libraries types (swift, obj-c) are displayed.

**Comment by matzew** [comment]
I mean more a download of ObjC bundle as well, so we could highlight, that we support both:
...Swift (zip)</a> 

...ObjC (zip)</a>

**Comment by corinnekrych** [comment]
all the links (download/view src/demo app etc..) could be declined in both version in the case of ios-push then. Maybe a tab (defaulted to Swift view) would make sense

**Comment by cvasilak** [comment]
yeap, [the page](https://aerogear.org/ios/) is Swift only with a logo on top to highlight and a  ``Notice``` though "looking for old AeroGear 1.x series?" where it points to our obj-c libraries (including obj-c Push links/downloads etc)

Regardless, for ios-push specifically I can have something like the following: 

Download (objc) (swift)
CocoaPods.org
Demo Application (objc) (swift)
View the Source (objc) (swift)
Learn More (objc) (swift)

I can do now if you think it's better. Guess when the 'AeroGear-Push' feature of the new web site is landed we can revisit it.

wdyt?

**Comment by corinnekrych** [comment]
Good point on the "Swift based iOS libraries" mention on top... I think we should rephrase it to "Swift first iOS libraries" and add a sentence to mention some of the lib comes in objc version.

+1 for the link (objc) (swift)

**Comment by cvasilak** [comment]
@matzew @corinnekrych as discussed I've updated with '(obj-c) (swift)' links.

**Comment by matzew** [comment]
obj-c ? Isn't it ObjC ? :)

**Comment by cvasilak** [comment]
@matzew done :)

**Comment by corinnekrych** [comment]
fine for me

**Comment by cvasilak** [comment]
rebased @matzew ready to merge ?

**Comment by matzew** [comment]
:+1:
